mkdocs-base.yml

```diff
@@ -168,4 +168,4 @@ nav:
       - Acknowledgments: reference/acknowledgements.md
 
   - Contribute:
-      - Contribution guidelines: api/contribute/contribute.md
+      - contribute.md
```

Comment:
Do we still need the other `contribute.md` located at https://github.com/percona/everest-doc/blob/main/docs/api/contribute/contribute.md?

Reply:
I dont think so. Catalaina has created the ToC. I will ask her to have a look.